lint-tasks.yml

```diff
@@ -103,7 +103,9 @@ tasks:
     cmds:
       - task: ":utils:clang-tidy"
         vars:
-          FLAGS: "--config-file={{.ROOT_DIR}}/.clang-tidy -p {{.CLP_FFI_PY_COMPILE_COMMANDS_DB}}"
+          FLAGS: >
+            --config-file={{.CLP_FFI_PY_CPP_SRC_DIR}}/.clang-tidy
+            -p {{.CLP_FFI_PY_COMPILE_COMMANDS_DB}}
           SRC_PATHS:
             ref: ".G_CPP_LINT_DIRS"
```

Comment:
How about (you suggested this a long time ago) setting this to only the files in which we've fixed all the clang-tidy issues? That way, we have a record of which files we've finished and we can enable automatic checking to ensure we're not breaking fixed files in future PRs.

Reply:
Sure